sparrow-py/pysrc/sparrow_py/_timestream.py

```diff
@@ -552,6 +552,17 @@ def is_not_null(self) -> Timestream:
         """
         return Timestream._call("is_valid", self)
 
+    def filter(self, condition: Timestream) -> Timestream:
+        """
+        Create a Timestream containing only the points where `condition` is `true`.
```

Comment:
Document Parameters too -- what is the `condition`

Reply:
Done